common/tree/script.go

```diff
@@ -806,7 +816,7 @@ func (d *CLTVMultisigClosure) Decode(script []byte) (bool, error) {
 	}
 
 	multisigClosure := &MultisigClosure{}
-	valid, err := multisigClosure.Decode(script[cltvIndex+2:])
+	valid, err := multisigClosure.Decode(tokenizer.Script()[tokenizer.ByteIndex():])
```

Comment:
would be better to put `tokenizer.Script()[tokenizer.ByteIndex():]` into a variable like:
```suggestion
	buf := tokenizer.Script()[tokenizer.ByteIndex():]
	valid, err := multisigClosure.Decode(buf)
```